keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/plugins/ArtifactSupplier.kt

```diff
@@ -39,7 +39,7 @@ interface ArtifactSupplier<A : DeliveryArtifact, V : VersioningStrategy> : Spinn
    */
   @JvmDefault
   fun publishArtifact(artifact: PublishedArtifact) =
-    eventPublisher.publishEvent(artifact)
+    eventPublisher.publishEvent(ArtifactPublishedEvent(listOf(artifact)))
```

Comment:
@gal-yardeni you kinda spotted this in the other PR but I didn't pay enough attention... 🤦